spec/features/user_views_past_orders_spec.rb

```diff
@@ -16,14 +12,43 @@
   end
 
   it "has an orders page for user" do
+    login_user(user)
+    visit '/profile'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.